mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -527,6 +528,10 @@ def match_log(self, line):
         if self.compare_log_idx < len(self.compare_log):
             regex = self.compare_log[self.compare_log_idx]
             # Either the line is matched as is or it is checked as a regular expression.
-            if regex in line or re.search(regex, line):
-                self.compare_log_idx += 1
+            try:
+                if regex in line or re.search(regex, line):
+                    self.compare_log_idx += 1
+            except error:
```

Comment:
Please see line https://github.com/ARMmbed/htrun/pull/146/files#diff-abaa341b8a118c20a730035041013df4R25
This is actually specific regex exception. I think the name "error" causes confusion.